frontend/front/src/api/apiSlice.js

```diff
@@ -108,7 +108,7 @@ export const apiSlice = createApi({
 
     /* Survey Endpoints */
     getSurveys: builder.query({
-      query: () => ({ url: "/surveys", method: "GET" }),
+      query: () => ({ url: "/surveys.json", method: "GET" }),
```

Comment:
I don't think adding `.json` was the correct fix since that's not an endpoint on the real back end, right? What was the problem here exactly?

Reply:
On `main`, clicking a home in a surveyor assignment encounters an error. I think this is because `/surveys` and `/surveys/:id` serve up HTML (see image)... Adding `.json` makes it work, but it's probably not the right fix.

So maybe this is an issue on the backend and not the frontend?

Reply:
Oh weird! We should probably bring this up with the back end team if this is how the real server responds (if you haven't already)

Reply:
I haven't. I'll squash and merge this then post on the backend Slack channel!